config/foreman.migrations/20220602164000_foreman_git_templates.yaml

```diff
@@ -0,0 +1 @@
+answers['foreman::plugin::git_templates'] ||= false
```

Comment:
The migration should be a `.rb` file, not `.yaml`. Same with the other file.

Reply:
Renamed it here and in all other pull requests (except the first one where I did it correct).